components/ai/code/ai-bbe/natural-programming-vscode-features.md

```diff
@@ -0,0 +1,15 @@
+---
+title: 'Ballerina Copilot: Bridging Requirements and Code with AI Precision'
+description: "Ballerina Copilot empowers developers to maintain perfect harmony between requirements and implementation through intelligent AI assistance. Seamlessly transform specifications into Ballerina code, preserve development intent, and ensure continuous alignment between your project's vision and its technical execution.  
+
+- **AI-Powered Code Generation** – Convert requirement specifications directly into production-ready Ballerina code using natural language templates
+
+- **Intent Preservation** – Automatically captures and stores user intention summaries from chat history, keeping critical project decisions documented
+
+- **Drift Detection & Repair** – Proactively identifies mismatches between documentation and source code, offering smart fixes via VS Code integration  
+
+- **Spec-Driven Testing** – Generate comprehensive test suites automatically from your requirement specifications  
+
```

Comment:
```suggestion
- **AI-Powered Code Generation** – Generate code against high-level business user requirements

- **AI-Powered Test Generation** – Generate Ballerina tests against business user requirements with just the APIs from the implementation, enabling identification of implementation deviation from the requirements

- **Drift Detection & Fixes** – Proactively identify drift between requirements, code, and documentation. Use smart fix suggestions via VS Code to review and accept AI-powered suggestions to address drift.  

- **Intent Preservation** – Maintain an effective summary of the prompts used to generate code, providing insight into the developers' input

```

Reply:
Have to switch the tabs too.

Can we keep drift check the third tab, but make it the default?